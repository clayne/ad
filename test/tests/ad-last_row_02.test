ad | | last_row-3.txt | 0
