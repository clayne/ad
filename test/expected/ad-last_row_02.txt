0000000000000000: 3031 3233 3435 3637 3839 4142 4344 450A  0123456789ABCDE.
----------------: (16 | 0x10)
0000000000000020: 3031 3233 3435 3637 3839 4142 4344 450A  0123456789ABCDE.
